UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -109,14 +109,15 @@ public async Task<SearchBatchResponse> GetBatchInfoBasedOnProducts(List<Products
             List<Products> internalNotFoundProducts = new List<Products>();
             var accessToken = await authFssTokenProvider.GetManagedIdentityAuthAsync(fileShareServiceConfig.Value.ResourceId);
             var productWithAttributes = GenerateQueryForFss(products);
-            var uri = $"/batch?limit={fileShareServiceConfig.Value.Limit}&start={fileShareServiceConfig.Value.Start}&$filter={fileShareServiceConfig.Value.ProductCode} {productWithAttributes}";
+            var uri = $"/batch?limit={fileShareServiceConfig.Value.Limit}&start={fileShareServiceConfig.Value.Start}&$filter={fileShareServiceConfig.Value.ProductCode} {productWithAttributes.Item1}";
 
             HttpResponseMessage httpResponse;
 
             string payloadJson = string.Empty;
             var productList = new List<string>();
             var prodCount = products.Select(a => a.UpdateNumbers).Sum(a => a.Count);
             int queryCount = 0;
+            logger.LogInformation(EventIds.FileShareServicePreparingToQueryENCFilesStart.ToEventId(), "Preparing file share service request for BatchId:{batchId} based on {sbLog} and _X-Correlation-ID:{CorrelationId}", batchId, productWithAttributes.Item2, correlationId);
```

Comment:
Log attribute `sbLog` is not much meaningful. Please can you revisit it.

Reply:
Yes updated.